libs/blocks/table/table.js

```diff
@@ -138,6 +141,17 @@ function handleExpand(e) {
  * @param {*} sectionParams that is from init()
  * @returns {boolean expandSection} that is the only variable get updated from sectionParams
  */
+
+function handleTitleText(cell) {
```

Comment:
I think the method should be placed before the docs on line 141, or after the `handleSection` method. The docs are not relevant for this method.

Reply:
@narcis-radu Have made the suggested changes.